tests/basic_commands_test.go

```diff
@@ -0,0 +1,56 @@
+// basic_commands_test.go
+package tests
+
+import (
+	"bytes"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/rs/zerolog/log"
+)
+
+func runL2CommandAndGetOutput(t *testing.T, cmdArgs ...string) string {
```

Comment:
If this is a utility method (looks like it), we should not taking in the testing object as a parameter. It confuses the reader.

In functions, as a rule, strive for minimum number of params.

For sophistication, *compose functions*

Reply:
Okay got it, I have a similar function in anohter file will add both in test utils file